time stamp  # Time stamp of stereo pair to save
uint32 id   # Node ID in the graph
